app/templates/style_studio.html

```
@@ -179,7 +179,15 @@ <h2 class="fw-bold mb-1">Pick a hairstyle</h2>
                             </div>
 
                             <div id="ai-panel-content" class="d-none align-items-center gap-3 w-100">
-                                <img id="ai-panel-thumb" src="" alt="" class="rounded-3 flex-shrink-0" style="width: 64px; height: 64px; object-fit: cover;">
+                                <div class="position-relative flex-shrink-0" style="width: 64px; height: 64px;">
+                                    <img id="ai-panel-thumb" src="" alt="" class="rounded-3 w-100 h-100" style="object-fit: cover;">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)